workflows/diagnostics/fv3net/diagnostics/offline/compute.py

```diff
@@ -74,6 +74,13 @@ def _get_parser() -> argparse.ArgumentParser:
         help="Local or remote path where diagnostic output will be written.",
     )
 
+    parser.add_argument(
+        "catalog_path",
```

Comment:
I think this should fix the remaining test failures:
```suggestion
        "--catalog_path",
```
In other words we need to make sure that the `catalog_path` argument is optional for backwards compatibility (see the relevant `argparse` documentation [here](https://docs.python.org/3/library/argparse.html#name-or-flags)).  This is based on looking at this error in [the unit tests](https://app.circleci.com/pipelines/github/ai2cm/fv3net/14687/workflows/3e091ab1-73b7-40c8-b191-2c3d2fc6ab5d/jobs/139190):

<details>
<p>

```
______________ test_offline_diags_integration[multiple_datasets] _______________

data_path = '/tmp/pytest-of-root/pytest-3/test_offline_diags_integration1'

    def test_offline_diags_integration(data_path):  # noqa: F811
        """
        Test the bash endpoint for computing offline diagnostics
        """
    
        batches_kwargs = {
            "needs_grid": False,
            "res": "c8",
            "timesteps_per_batch": 1,
            "timesteps": ["20160801.001500", "20160801.003000"],
        }
        trained_model = fv3fit.testing.ConstantOutputPredictor(
            input_variables=["air_temperature", "specific_humidity"],
            output_variables=["dQ1", "dQ2"],
        )
        trained_model.set_outputs(dQ1=np.zeros([19]), dQ2=np.zeros([19]))
        data_config = loaders.BatchesFromMapperConfig(
            loaders.MapperConfig(function="open_zarr", kwargs={"data_path": data_path},),
            **batches_kwargs,
        )
        with tempfile.TemporaryDirectory() as tmpdir:
            model_dir = os.path.join(tmpdir, "trained_model")
            fv3fit.dump(trained_model, model_dir)
            data_config_filename = os.path.join(tmpdir, "data_config.yaml")
            with open(data_config_filename, "w") as f:
                yaml.safe_dump(dataclasses.asdict(data_config), f)
>           compute_diags_args = compute._get_parser().parse_args(
                [
                    model_dir,
                    data_config_filename,
                    os.path.join(tmpdir, "offline_diags"),
                    "--evaluation-grid",
                    "c8_random_values",
                    "--n-jobs",
                    "1",
                ]
            )

workflows/diagnostics/tests/offline/test_compute.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/conda/envs/fv3net/lib/python3.8/argparse.py:1768: in parse_args
    args, argv = self.parse_known_args(args, namespace)
/opt/conda/envs/fv3net/lib/python3.8/argparse.py:1800: in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
/opt/conda/envs/fv3net/lib/python3.8/argparse.py:2034: in _parse_known_args
    self.error(_('the following arguments are required: %s') %
/opt/conda/envs/fv3net/lib/python3.8/argparse.py:2521: in error
    self.exit(2, _('%(prog)s: error: %(message)s\n') % args)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = ArgumentParser(prog='pytest.py', usage=None, description=None, formatter_class=<class 'argparse.HelpFormatter'>, conflict_handler='error', add_help=True)
status = 2
message = 'pytest.py: error: the following arguments are required: catalog_path\n'

    def exit(self, status=0, message=None):
        if message:
            self._print_message(message, _sys.stderr)
>       _sys.exit(status)
E       SystemExit: 2

/opt/conda/envs/fv3net/lib/python3.8/argparse.py:2508: SystemExit
```
</p>

</details>